app/controllers/admin/dashboard_controller.rb

```diff
@@ -1,6 +1,7 @@
 class Admin::DashboardController < Admin::BaseController
 
   def index
+    @weeks = Week.all.order(:created_at)
```

Comment:
i suggest creating a method on your `Week` model to extract this logic out of your controller. it should probably belong in the model rather than the controller... so you could call something like `Week.order_by_created_at`